### PR TITLE
Multiple enhancements to DLPAR, LPM and virt-net tests

### PR DESCRIPTION
**Pull request by narasimhan-v**
Handling lpar name via lparstat command since it is better.
The previous method of lsrsrc had a prerequisite of hostname to
be set properly, which was not consistent.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
Change inspired from https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1383

**Comment by narasimhan-v** [comment]
@smruti77 @abdhaleegit review please

File: io/net/virt-net/network_virtualization.py [hunk 1]
```diff
@@ -282,6 +296,11 @@ def test_add(self):
                 self.log.debug(output)
                 self.fail("lshwres fails to list Network virtualized device \
                            after add operation")
+            if mac not in output:
```
[the remark this answers is not in the document]
Comment (narasimhan-v) [reply io/net/virt-net/network_virtualization.py]: @smruti77 both mac and output are string, so will this not work ?
why str(output) is needed ?

[the remark this answers is not in the document]
Comment (smruti77) [reply io/net/virt-net/network_virtualization.py]: output is not in string

[the remark this answers is not in the document]
Comment (narasimhan-v) [reply io/net/virt-net/network_virtualization.py]: @smruti77 
https://github.com/avocado-framework-tests/avocado-misc-tests/blob/c2dc0f7251b4449ace537d5539429075672665fb/io/net/virt-net/network_virtualization.py#L614

[the remark this answers is not in the document]
Comment (narasimhan-v) [reply io/net/virt-net/network_virtualization.py]: This is a similar output variable, where I think it works. But fine, irrespective of that, it is safe to make it str(output), and will do that.

[the remark this answers is not in the document]
Comment (smruti77) [reply io/net/virt-net/network_virtualization.py]: @narasimhan-v  but that output is in list, you run that script after changing.
yesterday i checked but that one is falling.

[the remark this answers is not in the document]
Comment (narasimhan-v) [reply io/net/virt-net/network_virtualization.py]: @smruti77 force pushed, please review

**Comment by PraveenPenguin** [comment]
@smruti77 @Naresh-ibm thanks for ACK , merging

**Pull request merged**